services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/core/settings.py

```diff
@@ -43,6 +43,23 @@ class _BaseApplicationSettings(BaseApplicationSettings, MixinLoggingSettings):
         ),
     )
 
+    # TODO: this should be a common validator put in some common library and not here to allow reuse
```

Comment:
@giancarloromeo I would need your help here with this validator. It would have to be used quite a lot. Please read the comment.

Reply:
Sure, what comes in my mind is to reuse `field_validator`s this way: https://docs.pydantic.dev/2.0/usage/validators/#reuse-validators